_posts/2019-04-25_node_version_menager.md

```diff
@@ -0,0 +1,77 @@
+layout:    post
```

Comment:
Ej, ale po to mamy jekylla żeby sprawdzić jak ten post będzie wyglądał, a tutaj są takie błędy, że on się nawet nie pojawi na liście. Na wiki jest instrukcyja jak to odpalić :)
1. ten yaml powinien być pomiędzy dwoma separatorami '---'
2. w nazwie pliku powinny być myślniki a nie podkreślniki

Reply:
Coś czego mi mega brakuje w tym wpisie to linku do nvm :)

Reply:
dejm

Reply:
@krzysztof83 to nadal nie działa?

Reply:
działa, u mnie lokalnie już się pojawia z brancha

Reply:
link dodany, separatory w yamlu tez